mesheryctl/internal/cli/root/mesh/mesh.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Anirudh Jain
```

Comment:
`// Copyright 2020 Layer5 Authors`

or you can remove this license header altogether. This file is already covered at a project level by the Apache v2 license.